analytics_data_api/v0/views/learners.py

```diff
@@ -174,14 +174,21 @@ def get_queryset(self):
         """
         self._validate_query_params()
         query_params = self.request.QUERY_PARAMS
+
+        # Ordering by problem_attempts_per_completed can be ambiguous because
+        # it's a ratio values could be infinite (e.g. divide by zero) if no problems
```

Comment:
This wording is confusing. Looks like you might have forgotten to delete "it's a ratio"?